### PR TITLE
number of integer and binary variables are also retrieved

### PR DESCRIPTION
**Pull request by ozancakiroglu95**

### CODE REVIEW

File: src/pyscipopt/scip.pyx [hunk 1]
```diff
@@ -1686,6 +1686,14 @@ cdef class Model:
         """Retrieve the number of constraints."""
         return SCIPgetNConss(self._scip)
 
+    def getNIntVars(self):
+        """Retrieve the number of integer variables"""
```
Comment (fserra) [inline src/pyscipopt/scip.pyx:1690]:
```suggestion
        """gets number of integer active problem variables"""
```

File: src/pyscipopt/scip.pyx [hunk 2]
```diff
@@ -1686,6 +1686,14 @@ cdef class Model:
...
+        return SCIPgetNIntVars(self._scip)
+
+    def getNBinVars(self):
+        """Retrieve the number of binary variables"""
```
Comment (fserra) [inline src/pyscipopt/scip.pyx:1694]:
```suggestion
        """gets number of binary active problem variables"""
```

**Review by fserra — CHANGES_REQUESTED** [review]
thanks!

**Review by fserra — APPROVED** [review]

**Comment by fserra** [comment]
thanks for your contribution

**Comment by ozancakiroglu95** [comment]
I appreciate your interest. Thank you, too..

**Pull request merged**